components/OAuth2.0/OauthHttpLocalServer.c

```diff
@@ -102,26 +101,26 @@ static esp_err_t Spotify_HttpsCallbackHandler(httpd_req_t *HttpdRequest)
 }
 
 /**
- * this strcut is http URL handler if receive "/" Spotify_RequestDataAccess getting run
+ * this strcut is http URL handler if receive "/" RequestDataAccess getting run
  */
-static const httpd_uri_t Spotify_Request_Access_URI = {
+static const httpd_uri_t Request_Access_URI = {
     .uri = "/",
     .method = HTTP_GET,
-    .handler = Spotify_RequestDataAccess};
+    .handler = RequestDataAccess};
 
 /**
  * this strcut is http URL handler if receive "/callback" HttpsUserCallBackFunc getting run
  */
-static const httpd_uri_t Spotify_Response_Access_URI = {
+static const httpd_uri_t Response_Access_URI = {
     .uri = "/callback/",
```

Comment:
the `uri` should also be need to be configurable

Reply:
done.